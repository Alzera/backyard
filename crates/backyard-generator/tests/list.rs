use backyard_generator::generate;
use backyard_parser::parse_eval;

#[test]
fn basic() {
  let arena = bumpalo::Bump::new();
  let asts = parse_eval(&arena, "list($a, $b) = [0, 1];").unwrap();
  insta::assert_yaml_snapshot!(generate(&asts).unwrap());
}
