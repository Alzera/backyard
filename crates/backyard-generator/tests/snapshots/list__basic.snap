---
source: crates/backyard-generator/tests/list.rs
expression: generate(asts).unwrap()
snapshot_kind: text
---
"list($a, $b) = [0, 1];"
